### DISCUSSION TITLE
Custom Delimiters not working inside of v-for

### DISCUSSION THREAD

**Discussion by ghost** [vuejs/petite-vue#108] [Q&A]
I am looping over a array of objects ( ``` [ id: 1, value: 1 ]  ```) & i  have a setup petite vue with custom Delimiters to use single curly braces ``` { value } ``` with ``` $delimiters: ['{', '}'] ```, but it does not work inside of v-for needs to be double curly braces ``` {{ value }} ```.

`createApp({ $delimiters: ['{', '}'] }).mount()`

how i can fix these problem or it will be fixed in upcoming versions of petite vue.

I would like to say Evan, you really did an awesome job, petite vue is really reactive, performant, & awesome. So, Thanks a lot.

for the future, try to extend it with router, supsense, portal (teleport), transition & more...

**Comment by halivert** [comment]
There are two pull request related to your question (#100, #90). Maybe one of those fixes would be part of the next update of petite-vue, but in the meantime, you could use [this solution](https://github.com/vuejs/petite-vue/pull/90#issuecomment-980793131).

**Reply by ghost** [reply]
just i need to list things to be fixed, because i think petite vue has a lot more power.
